test/Microsoft.Dnx.Compilation.CSharp.Tests/FakeCompilerOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Microsoft.Dnx.Compilation.CSharp.Tests
+{
+    class FakeCompilerOptions : ICompilerOptions
```

Comment:
nit: add the modifier - I think we are adding them always even if they are the same as default. Same for the const strings in the `RoslynCompilerTest`